homeworks/alexandr-shmalenko/homework2/java4.js

```diff
@@ -0,0 +1,26 @@
+const threeDigitNumber = +prompt('Введіть тризначне число');
+const firstDigit1 = Math.floor(threeDigitNumber / 100);
```

Comment:
```suggestion
const firstDigit = Math.floor(threeDigitNumber / 100);
```
You don't need 1 in the end, useless information, what does it mean?